src/aioamazondevices/api.py

```
@@ -260,13 +260,19 @@ async def _get_devices_endpoint_data(self) -> dict[str, dict[str, Any]]:
-        _, raw_resp = await self._http_wrapper.session_request(
-            HTTPMethod.GET,
-            url=f"https://alexa.amazon.{self._session_state_data.domain}{URI_NOTIFICATIONS}",
-        )
+        try:
+            _, raw_resp = await self._http_wrapper.session_request(
+                HTTPMethod.GET,
+                url=f"https://alexa.amazon.{self._session_state_data.domain}{URI_NOTIFICATIONS}",
+            )
+        except CannotRetrieveData:
+            _LOGGER.warning(
+                "Failed to obtain notification data.  Timers and alarms have not been updated"  # noqa: E501
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)